JobRecommendationService.md

```diff
@@ -3,120 +3,68 @@
 
 - [Summary](#summary)
 - [Request structure](#request-structure)
-  - [JobData](#jobdata)
-    - [Job](#job)
-  - [UserData](#userdata)
-  - [UserPreferences](#userpreferences)
-    - [PreferredLocation](#preferredlocation)
 - [Response Structure](#response-structure)
 - [Versioning](#versioning)
 
 ----------------------------
 # Summary
 
-Given a job ID and, optionally, an email, and a set of user preferences, the Job Recommendation Service provides recommendations of relevant and similar jobs.
+Given a job ID and, optionally, a algorithm name and a host site, the Job Recommendation Service provides recommendations of relevant and similar jobs.
 
 ----------------------------
 # Request structure
-The service supports GET and POST requests to:
-https://api.careerbuilder.com/core/recommendations/job.
+The service supports `GET` and `POST` requests to:
+https://api.careerbuilder.com/consumer/recommendations/platform.
 
 OAUTH credentials are **required** and Documentation for CareerBuilder authentication is provided [here](https://apimanagement.cbplatform.link/#/oauth/faq) by the Platform Software team.
 
-A request is composed of 3 main parts:
-`job_data`, `user_data`, `user_preferences`.
+The request parameters:
 
 | Param    | Type | Required | Description |
 |----------|------|----------|-------------|
-| `job_data` | [**JobData**](#jobdata) | **TRUE** | **JobData** contains a list of jobs which are needed for recommendations.
-| `hostsite` | String | **False** | Hostsite for `source_jobs`. Currently supports US, UK, CA. Defaults to US if omitted 
-| `user_data` | [**UserData**](#userdata) | **FALSE** | **UserData** contains the  email address for a given user. This data is used for filtering jobs that the user has already applied to from the final recommendations set.
-| `user_preferences` | [**UserPreferences**](#userpreferences) | **FALSE** | **UserPreferences** indicates preferences of the user including location, compensation, and job titles.
-
-### JobData
-| Param    | Type | Required | Description |
-|----------|------|----------|-------------|
-| `source_jobs` | List<[**Job**](#job)> | **TRUE** | A list of **Job**.
-
-Note: currently only the first job in the list is used for recommendations.
-
-#### Job
-| Param    | Type | Required | Description |
-|----------|------|----------|-------------|
-| `job_did` | String | **TRUE** | This needs to be a valid jobdid.
-
-### UserData
-**UserData** contains only one String parameter: `email_address`.
-
-| Param    | Type | Required | Description |
-|----------|------|----------|-------------|
-| `email_address` | String | **TRUE** | The email of the registered user.
+| `jobID` | String | **True** | This needs to be a valid jobdid.
```

Comment:
Before the job recommendation workflow is frontend -> visitor-recs-js -> JRS. After migration the workflow become frontend -> JRS. Since this frontend use camelcase and we try to avoid asking frontend team to update frontend, we have to use camelcase.

Reply:
Got it. Thanks :)